### ISSUE TITLE
how do retrive current viewport and scissor box.

### ISSUE THREAD

**Issue by Hperigo** [grovesNL/glow#146]
Hello, I´m writing a glow-imgui renderer and in the official ImGui OpenGl backend there´s a specific part of the code that retrieves the current GL_VIEWPORT and GL_SCISSOR_BOX. See here: 
[https://github.com/ocornut/imgui/blob/master/backends/imgui_impl_opengl3.cpp#L339](https://github.com/ocornut/imgui/blob/master/backends/imgui_impl_opengl3.cpp#L339)

I know that glow provides `gl.get_parameter_i32`, but that only returns a single i32, is there a way to retrieve an array from it? 

thanks

**Comment by Hperigo** [comment]
ok,  that was not that hard, but it took me a little bit of time figure this out:
```
    unsafe fn get_parameter_slice_i32(&self, parameter : u32, out : &mut[i32] ){
        let gl = &self.raw;
        gl.GetIntegeri_v( parameter, 0, &mut out[0] );
    }
```
@grovesNL would you accept a pull request with this function? Do you need more similar implementation of glGet?
for eg: `get_parameter_slice_f32`?

**Comment by grovesNL** [comment]
Yes, a PR to add that would be great! We could start with `i32`, `f32`. Maybe we could also call it `get_parameter_i32_slice` and `get_parameter_f32_slice` to be consistent with some of the other functions.

We can also copy `get_uniform_i32` and `get_uniform_f32` for the web backend e.g. filling `&mut [f32]` from `Float32Array`).

**Comment by grovesNL** [comment]
Fixed by #147, thanks @Hperigo!
